File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/DateTimePicker/Programming/ValueChangedEventPage.vb
Namespace CompanionKit.Controls.DateTimePicker.Programming

	Public Class ValueChangedEventPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles Load event of the example' UserControl.
        ''' </summary>
        Private Sub ValueChangedEventPage_Load(sender As Object, e As EventArgs) Handles Me.Load
            ' Set initial value for text of the label that represents current value of the DateTimePicker.
            Me.mobjSelectedValueLabel.Text = String.Format("You selected:" & vbCr & vbLf & "{0}", Me.mobjDemoDateTimePicker.Value.ToLongDateString())
        End Sub


        ''' <summary>
        ''' Handles ValueChanged event of the demonstrated DateTimePicker.
        ''' </summary>
        Private Sub mobjDemoDateTimePicker_ValueChanged(sender As Object, e As EventArgs) Handles mobjDemoDateTimePicker.ValueChanged
            ' Update text of the label that represents current value of the DateTimePicker.
            Me.mobjSelectedValueLabel.Text = String.Format("You selected:" & vbCr & vbLf & "{0}", Me.mobjDemoDateTimePicker.Value.ToLongDateString())
        End Sub
    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/DateTimePicker/Programming/ValueChangedEventPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.DateTimePicker.Programming
{
    public partial class ValueChangedEventPage : UserControl
    {
        public ValueChangedEventPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Load event of the example' UserControl.
        /// </summary>
        private void ValueChangedEventPage_Load(object sender, EventArgs e)
        {
            // Set initial value for text of the label that represents current value of the DateTimePicker.
            this.mobjSelectedValueLabel.Text = string.Format("You selected:\r\n{0}", this.mobjDemoDateTimePicker.Value.ToLongDateString());
        }


        /// <summary>
        /// Handles ValueChanged event of the demonstrated DateTimePicker.
        /// </summary>
        private void mobjDemoDateTimePicker_ValueChanged(object sender, EventArgs e)
        {
            // Update text of the label that represents current value of the DateTimePicker.
            this.mobjSelectedValueLabel.Text = string.Format("You selected:\r\n{0}", this.mobjDemoDateTimePicker.Value.ToLongDateString());
        }
    }
}